docs/getting-started/connect-widget.mdx

```diff
@@ -0,0 +1,91 @@
+---
+title: "Connect Widget"
+description: "Overview of the configurations for the widget."
+---
+
+# Link the the Metriport Connect widget in your app
+
+With Metriport, you get a pre-built [Connect widget](https://connect.metriport.com/?token=demo)
+that you can easily plug into your app, so that your users can connect all of their data sources to your application!
+Here's a preview:
+
+<iframe
+  width="90%"
+  height="700"
+  src="https://connect.metriport.com/?token=demo"
+>
+  {" "}
+</iframe>
+<br />
+
+# Url Paramaters
+
+Set url params to validate the current session of the widget and make
+certain customizations to fit your product needs.
+
+### token
+
+This is the token generated by metriport. Check the quick start for more details.
+
+```typescript
+`https://connect.metriport.com/?token=${connectToken}`;
+```
+
+### colorMode
+
+You can update the color mode to be either light or dark.<br/>
+
+```typescript
+`https://connect.metriport.com/?token=${connectToken}&colorMode=dark`;
+```
+
+<iframe
+  width="90%"
+  height="700"
+  src="https://connect.metriport.com/?token=demo&colorMode=dark"
+>
+  {" "}
+</iframe>
+
+### customColor
+
+You can set your own color as the highlight color by adding a default color such as orange or add a hex code.<br/>
+
+```typescript
+`https://connect.metriport.com/?token=${connectToken}&customColor=orange`;
+```
+
+<Tip>
+  If you wish to add a hex code ensure that it is URL encoded ie.
+  customColor=%23FF5733
+</Tip>
+
+<iframe
+  width="90%"
+  height="700"
+  src="https://connect.metriport.com/?token=demo&customColor=orange"
+>
+  {" "}
+</iframe>
+
+### providers
+
+You can choose the providers you wish to display to your users.<br/>
+
+```typescript
+`https://connect.metriport.com/?token=${connectToken}&providers=fitbit,oura,garmin`;
+```
+
+<Tip>
+  This defaults to all providers if no param is set. Once you set the provider
+  paramater you will need to manually add providers or it will default to your
+  last request.
+</Tip>
```

Comment:
Let's explicitly state that it's a comma-separated list, and we need to link the dev to what the available provider options are.

For now maybe we can link to https://docs.metriport.com/more-info/our-integrations , and add a column called Internal Name or something?